### PR TITLE
Updated translation

### PR DESCRIPTION
**Pull request by monjo44**

### CODE REVIEW

File: addons/Infractions/language.php [hunk 1]
```diff
@@ -9,28 +9,28 @@
...
 
 $infractions_language = array(
-	'infractions' => 'Infractions',
+	'infractions' => 'Bestrafungen',
 	'infractions_icon' => '', // Icon to display before "Infractions" in navbar
 	'view' => 'View &raquo;',
-	'staff_member' => 'Staff Member',
-	'action' => 'Action',
-	'actions' => 'Actions',
+	'staff_member' => 'Team Mitglied',
```
Comment (Cl1608Ho) [inline addons/Infractions/language.php]: "Teammitglied" without the whitespace would fit better

File: Nameless [hunk 2]
```diff
@@ -0,0 +1 @@
+Subproject commit 42d713fcc0e4899698b2d2dd27320f9bcd648f7d
```
Comment (Cl1608Ho) [inline Nameless]: What's that? :o

**Comment by partydragen** [comment]
Why you removed
-	'action' => 'Action',
-	'actions' => 'Actions',

and changed Staff Member to another thing when this is english file? Is it a another language on the english file?

File: Nameless (hunk 2 above)
Comment (monjo44) [inline Nameless]: Ehm I dont know. This was my first pull request ever :D

**Comment by monjo44** [comment]
This is a german translation for the Infraction addon. Thought you coult use it somehow :)

File: addons/Infractions/language.php (hunk 1 above)
Comment (BukkitTNT) [inline addons/Infractions/language.php]: You deleted action & actions

Comment (monjo44) [inline addons/Infractions/language.php]: Its not deletet.. Its translated.. check out the rest of the file :)

**Comment by samerton** [comment]
Thanks for this, instead of overwriting the existing language I've moved this to a separate file (giving you full credit of course).

**Pull request closed without merging**